ios/ElectrodeReactNativeBridge/ElectrodeBridgeHolder.m

```diff
@@ -21,6 +21,43 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+@interface ElectrodeQueuedEventListener: NSObject
+@property (nonatomic, strong) NSUUID *uuid;
+@property (copy) ElectrodeBridgeEventListener listener;
```

Comment:
Proposal: having a property with both atomic and copy seems a bit unusual, so I would propose writing the "atomic" here, too, to document that this was done on purpose.

Reply:
@jomnius Thanks for the comments. Please check apple docs here on block property example: https://developer.apple.com/library/content/documentation/Cocoa/Conceptual/ProgrammingWithObjectiveC/WorkingwithBlocks/WorkingwithBlocks.html